public/index.html

```diff
@@ -31,6 +31,9 @@
 integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
  crossorigin="anonymous"
 />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
```

Comment:
Is there any special reason why we can't have this inside of our CSS/SASS? 

When you do it this way you're going to have to fetch these resources almost always but when you add it to your CSS then you build it once and render all the time.